#!/usr/bin/env python
PACKAGE = "motor"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("servo_offset", int_t, 0, "Servo Offset", 1000, 0, 4095)
gen.add("servo_gain", double_t, 0, "Servo Gain", -13.0, -100, 100)

exit(gen.generate(PACKAGE, "motor_node", "motor"))
